src/module/item/consumable/data.ts

```diff
@@ -43,7 +43,17 @@ interface ConsumableSystemData
         Omit<PhysicalSystemData, "traits"> {
     stackGroup: AmmoStackGroup | null;
 }
-type SourceOmission = "bulk" | "description" | "hp" | "identification" | "material" | "price" | "temporary" | "usage";
+
+type SourceOmission =
+    | "apex"
+    | "bulk"
+    | "description"
+    | "hp"
+    | "identification"
+    | "material"
+    | "price"
+    | "temporary"
+    | "usage";
```

Comment:
Can save a few lines by just putting `apex?: never` in both system source and system data (likewise with treasure)